CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
  * Added support for AsyncHttpClient 2.x
  * Added [`global_labels`](https://www.elastic.co/guide/en/apm/agent/java/current/config-core.html#global-labels) configuration option.
    This requires APM Server 7.2+.
+ * Added basic support for JMS- distributed tracing for basic scenarios of `send`, `receive`, `receiveNoWait` and 
+   `onMessage`. Both Queues and Topics are supported. Async `send` APIs are not supported in this version. 
+   NOTE: This feature is currently marked as "Experimental" and is disabled by default. In order to enable, it is 
```

Comment:
marked as `incubating`?

Reply:
Let's also discuss whether we can enable this by default but still having it marked as incubating. Do you think that the instrumentation and context handling is safe enough? What's missing that we can confidently remove the `incubating` tag?

Reply:
I think the instrumentation is safe enough, however there are some stuff that we may want to change following the general messaging discussion, which will be kinda breaking changes, mainly the new `transaction.type` (I added `messaging`, but this may change).
In addition, `span.type` may change, as well as the span names (maybe we will want to remove the destination name from the span name once we add it to the schema).

Lastly, there is some UI handling required to fix some scenarios like very delayed receiver span, or a receiver span/transaction starting before (or a lot before) the sender span. As long as these are not implemented, they may cause a problematic user experience, so better start off with `incubating` I think.